lib/aurora_bootstrapper/notifier.rb

```diff
@@ -12,7 +12,59 @@ def export_date
     
     # ENV is string to string dictionary
     def export_date_override
-      DateTime.now.strftime("%Y-%m-%d") if ENV.key?('EXPORT_DATE_OVERRIDE')
+      datetime_subfolder_str = nil
+
+      if ENV.key?('EXPORT_DATE_OVERRIDE')
+        now = Date.today
+        
+        # expiration time is 30 days
+        for i in 1..30
+          datetime_subfolder_str = check_db_dump_done_for_one_day( datetime: (now-i))
```

Comment:
this variable name is confusing to me because i see `subfolder_str`, but expect the function to return a date. Can we name it something that indicates its function rather than how we derive it - `date_override` instead of `datetime_subfolder_str`

Reply:
Sure